loadgenerator/loadgen

```diff
@@ -458,6 +496,13 @@ def _get_parser():
             description='Show the status of all load generator jobs whether running or not')
     subparser.set_defaults(func=status)
 
+    subparser = subparsers.add_parser('autorun',
+            help='Create load generator with Cloud Run',
+            description='automatically starts a load generation job for a target address with Cloud Run')
+    subparser.add_argument('target_address', type=str,
+            help='The target address (without http/s prefix) to send laod to')
```

Comment:
It would be great if we could find this automatically instead of expecting the user to know it. Levi did that [here](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/dd3341773b4c3aa60926d06b2e0d18b703d48321/tests/provisioning/provision_test.py#L58). You also may need to authenticate kubectl first to talk to the cluster, which he did [here](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/dd3341773b4c3aa60926d06b2e0d18b703d48321/tests/provisioning/provision_test.py#L122)

Reply:
awesome, I'll try to do that instead